src/psadt-silent.template/templates/tools/chocolateyuninstall.ps1

```diff
@@ -0,0 +1,13 @@
+﻿$ErrorActionPreference = 'Stop' # stop on all errors
+$toolsDir   = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$destination = (Join-Path $toolsDir -ChildPath $env:ChocolateyPackageName)
+
+
+$adtExe = Join-Path $destination -ChildPath "Deploy-Application.exe"
+
+$packageArgs = @{
+    ExetoRun = $adtExe
+    statements = "-DeploymentType 'Uninstall' -DeployMode 'Silent'"
+}
+
+Start-ChocolateyProcessAsAdmin @packageArgs
```

Comment:
I'd suggest switching to `Uninstall-ChocolateyPackage` if possible, as that has support for valid exit codes.